javascript/javascript2/week3/test.js

```diff
@@ -0,0 +1,98 @@
+//Warmup
+//1. log out text in 2.5 sec
+setTimeout(() => {
+  console.log("Called after 2.5 sec");
+}, 2500);
+
+//2.Create a function that takes 2 parameters: delay and stringToLog. Calling this function should log out the stringToLog after delay seconds. Call the function you have created with some different arguments.
+function delayFunction(delay, stringTolog) {
+  setTimeout(() => {
+    console.log(stringTolog);
```

Comment:
While this certainly solves the task, it doesn't use promises, which was probably the intention. So if you want to practice using promises, you can update it to do that 😉

You would resolve the promise instead of executing the log immediately. Then you can execute `console.log` when your promise has resolved, either by using a promise-chain (`.then(...)`) or async/await.

Same or similar advice is of course true for all exercises 🙂